assets/images/about/background_dsk_1.svg

```diff
@@ -0,0 +1 @@
+<svg height="444.755906pt" preserveAspectRatio="none" viewBox="0 0 960.009449 444.755906" width="960.009449pt" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><clipPath id="a"><path d="m0 0h959.980469v444.726562h-959.980469zm0 0"/></clipPath><clipPath id="b"><path d="m0 0h959.980469v338h-959.980469zm0 0"/></clipPath><clipPath id="c"><path d="m0 337h959.980469v107.726562h-959.980469zm0 0"/></clipPath><g clip-path="url(#a)"><path d="m0 0h959.980469v444.730469h-959.980469zm0 0" fill="#fff" fill-rule="evenodd"/></g><g clip-path="url(#b)"><path d="m480.359375 337.492188h-480.359375v-337.492188h960.71875v337.492188zm0 0" fill="#f4f4f4" fill-rule="evenodd"/></g><g clip-path="url(#c)"><path d="m-56.238281 417.855469 58.308593-8.929688c58.335938-9.070312 175.007813-26.644531 291.683594-24.546875 116.644532 2.070313 233.320313 24.691406 349.992188 37.957032 116.648437 13.519531 233.320312 17.714843 291.660156 20.097656l58.335938 2.210937v-107.152343h-58.335938c-58.339844 0-175.011719 0-291.660156 0-116.671875 0-233.347656 0-349.992188 0-116.675781 0-233.347656 0-291.683594 0h-58.308593zm0 0" fill="#f4f4f4" fill-rule="evenodd"/></g></svg>
```

Comment:
Here and below if you see the red warning icon in the PR, please add a new line.

Reply:
done